cumulusci/tasks/salesforce/ActivateFlowProcesses.py

```diff
@@ -0,0 +1,46 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+from cumulusci.core.utils import process_list_arg
+import click
+
+
+class ActivateFlowProcesses(BaseSalesforceApiTask):
+    """
+    Activate the Flows with the supplied Developer Names
+    """
+
+    def _init_options(self, kwargs):
+        self.options = {}
+        self.options["developer_names"] = process_list_arg(
+            self.task_config.options["developer_names"]
+        )
+
+    api_version = "43.0"
```

Comment:
It makes sense to specify a particular API version that we know has the FlowDefinition object, but why such an old one?

Reply:
So I think @davidmreed  had mentioned that there was a specific reason we were testing with version 43.0 but I could be making this up was a little bit ago.